docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -104,9 +119,9 @@ The [Get a Connection](/docs/apps-api/tax-app-connection#get-connection-status)
 
 ## Enabling tax providers in the control panel
 
-Once the tax provider's app has been successfully installed and basic authentication credentials have been provided via the Update a Connection request, users will be able to enable the tax provider on all supplied merchant and/or partner test stores provided by the tax provider in [sharing provider details with BigCommerce](#sharing-provider-details-with-bigcommerce).
+Once the tax provider's app has been successfully installed and basic authentication credentials have been provided via the Update a Connection request, merchants and partners will be able to enable the tax provider on all supplied merchant and/or partner test stores provided by the tax provider in [sharing provider details with BigCommerce](#sharing-provider-details-with-bigcommerce).
 
-To enable the tax provider, users must navigate to **Settings > Setup > Tax** in an active MSF-enabled BigCommerce control panel and click **Enable** next to the associated tax provider.
+To enable the tax provider, merchants must navigate to **Settings > Setup > Tax** in an active MSF-enabled BigCommerce control panel and click **Enable** next to the associated tax provider.
```

Comment:
Minor question, should this be changed to merchants or **both** merchants and partners? (i.e. do partner stores have the same UI as merchant stores?)

Reply:
Got my answer [here](https://github.com/bigcommerce/dev-docs/pull/1728#discussion_r1152568576)